packages/kit/src/exports/index.js

```diff
@@ -43,3 +43,13 @@ export function json(data, init) {
 		headers
 	});
 }
+
+/**
+ * Generates a `ValidationError` object.
+ * @param {number} status
+ * @param {Record<string, any>} errors
+ * @param {FormData} [form]
+ */
+export function validation(status, errors, form) {
```

Comment:
what about `throw invalid(...)`? I think that better communicates 'something went wrong'

```suggestion
export function invalid(status, errors, form) {
```

Reply:
yeah that's better